README.md

```diff
@@ -96,13 +96,9 @@ Then elsewhere in your code:
 var Plotly = require('./path/to/custom-plotly');
 ```
 
-#### Non-ascii characters
+Alternatively you could browserify a custom bundle of desired trace modules e.g. `pie` and `choropleth` using
+`npm run partial-bundle pie choropleth name=custom`
```

Comment:
it would be nice if this followed the usual command-line flag patterns and hence had `--name=custom` instead of just `name=custom` IMO

Reply:
As a side note I don't want to rely on [minimist](https://www.npmjs.com/package/minimist); but OK I could look for `--name` when parsing the arguments.

Reply:
Done in https://github.com/plotly/plotly.js/pull/5508/commits/8d902aadcb66dccf945101584b3b202a350cc5a9.